firebase-auth/src/androidMain/kotlin/dev/gitlive/firebase/auth/auth.kt

```diff
@@ -118,9 +118,26 @@ public actual class FirebaseAuth internal constructor(internal val android: com.
 
 public val AuthResult.android: com.google.firebase.auth.AuthResult get() = android
 
-public actual class AuthResult internal constructor(internal val android: com.google.firebase.auth.AuthResult) {
+public actual class AuthResult(internal val android: com.google.firebase.auth.AuthResult) {
     public actual val user: FirebaseUser?
         get() = android.user?.let { FirebaseUser(it) }
+    public actual val credential: AuthCredential?
+        get() = android.credential?.let { AuthCredential(it) }
+    public actual val additionalUserInfo: AdditionalUserInfo?
+        get() = android.additionalUserInfo?.let { AdditionalUserInfo(it) }
+}
+
+public actual class AdditionalUserInfo(
+    public val android: com.google.firebase.auth.AdditionalUserInfo,
```

Comment:
just need this to follow the same pattern as the others (internal variable with public extension property)